src/main/java/com/redhat/devtools/intellij/tektoncd/actions/component/CreateClusterTaskAction.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
```

Comment:
I remember I had once the comment on the package name which seems to be herited from the OpenShift plugin so I think it should be renamed